### PR TITLE
initial tests for the sensitive banner

### PR DESCRIPTION
**Pull request by WMDE-Fisch**

### CODE REVIEW

**Comment by wmde-manicki** [comment]
I had a look and it looks really good for me. I have run sensitive-banner-related tests and only one failed, and one was skipped due to some not implemented step - but these are understandable as functionalities those tests refer to are actually missing from the banner (clearing bank data while switching a payment method and submitting the form by pressing ENTER).
Just thinking why is this marked as "work in progress"? Because of those two tests still not passing? Or are you going to add some "general logic" to those tests before it is ready?
I'd say this is almost done :)

File: features/sensitive.ui.feature [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    And I click sensitive banner debit option
+    Then The sepa donation part should be visible
+    And The nonsepa donation part should not be visible
+    And The person donation part should be visible
+    And The company donation part should not be visible
+    And The address donation part should be visible
+    And The next button should be visible
+    And The finish donation button should not be visible
+
+  Scenario Outline: Checks if the non-deposit payment methods open correctly
```
Comment (wmde-manicki) [inline features/sensitive.ui.feature]: Shouldn't it say `non-**debit** payment methods` here? One of payment_methods listed in examples is indeed "deposit". And if this comment is right, line 17 above should then say `Checks if the debit payment`, I guess.

**Comment by WMDE-Fisch** [comment]
Hmm its WIP because I will add even more tests. But true it could be merged already.I will just fix the naming in the one test. :-)

**Comment by wmde-manicki** [comment]
Both ways (merging now or when ALL the tests are here) are absolutely fine for me. Just thought then having it merged now would possibly make expanding test suite(s) easier (smaller PRs to review, for instance).

**Comment by WMDE-Fisch** [comment]
ready to merge :-)

**Comment by wmde-manicki** [comment]
Done, sir

**Comment by WMDE-Fisch** [comment]
Thanks, captain!

**Pull request merged**